exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39333102-7652-4699-ba0e-d184aeec1753",
+   "metadata": {},
+   "source": [
+    "Before we start we need to configure the script-languages directory and flavor. See [Configure SLC repository](./configure_slc_flavor.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25604520-c997-4d2b-8138-02a64e7231d0",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "### Open Secure Configuration Storage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da57e77a-b41f-45e3-9a17-4b474bba898b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%run ../utils/access_store_ui.ipynb\n",
+    "display(get_access_store_ui('../'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81fbc9a1-718a-4d93-8753-85b032998c1b",
+   "metadata": {},
+   "source": [
+    "### Instantiate SLCT Manager\n",
+    "\n",
+    "We create an instance of the SlctManager class from the notebook connector.\n",
+    "This class has some utility function which simplifies the use of the `exaslct` API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "27352b75-823d-40a1-b8df-d33218fd7de6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from exasol.nb_connector import slct_manager\n",
+    "slctmanager = slct_manager.SlctManager(ai_lab_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9fba4c9e-4a1b-4f84-a9a6-f05810aa8922",
+   "metadata": {},
+   "source": [
+    "### Import some utility functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c554d698-d5a2-4f9b-846a-f606a3f183b7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from exasol.nb_connector.utils import upward_file_search\n",
+    "\n",
+    "# This NB may be running from various locations in the NB hierarchy.\n",
+    "# Need to search for the styles NB from the current directory upwards.\n",
+    "\n",
+    "%run {upward_file_search('utils/ui_styles.ipynb')}\n",
+    "%run {upward_file_search('utils/popup_message_ui.ipynb')}\n",
+    "%run ./utils/file_system_ui.ipynb\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c88668d2-83e5-4812-a05c-936912068f03",
+   "metadata": {},
+   "source": [
+    "## Use flavor as is\n",
+    "\n",
+    "Currently used flavor is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "617380fa-6b14-40a5-bffd-ca8fed37950c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slct_manager.REQUIRED_FLAVOR"
```

Comment:
From the perspective of the tutorial, `REQUIRED_FLAVOR` sound awkward.
As far as I did understand, the user first selects a default flavor and now wants to build an SLC based on this flavor. If this is correct, then a term like `slct_manager.SELECTED_FLAVOR` or just `slct_manager.FLAVOR` would be more intuitive, IMHO.